src/utils/__mocks__/worker-factory.ts

```diff
@@ -9,7 +9,7 @@ class NodeWorker {
   onmessage: ((event: MessageEvent) => unknown) | null;
 
   constructor(file: string) {
-    const worker = new Worker(__dirname + '/node-worker-contents.js', {
+    const worker = new Worker(__dirname + '/node-worker-contents.mjs', {
```

Comment:
Oh, interesting. #5556 touches this code too but this one should land first.

Reply:
Oops, I wasn't aware that it was touching it. Just saw it. I had to convert this to .mjs so it understands that it's an ES module file. I'm not so sure if that works when we add it as an eval string. But probaby it's fine to keep it as `require` if it's only kept as a string as eslint won't complain :)